app/views/shelters/show.html.erb

```diff
@@ -0,0 +1,26 @@
+<style media="screen">
+  input[type="submit"] {
+    background: none;
+    border: none;
+    outline: none;
+    text-decoration: underline;
+    color: blue;
+    cursor: pointer;
+  }
+</style>
```

Comment:
I included this CSS to make the submit button styled to look like a link because the user story specified clicking a link to delete.
I am considering moving my CSS into a separate file as I continue to add more.